src/Microsoft.Tye.Hosting/Dashboard/Pages/ServiceDetails.razor

```diff
@@ -1,5 +1,6 @@
 ﻿@page "/services/{ServiceName}"
 @inject Application application
+@inject HttpClient Http
```

Comment:
See my other comment regarding `HttpClient` injection. Can you try to see if you can get away with creating a `HttpClient` instance directly?